elyra/api/elyra.yaml

```diff
@@ -304,20 +304,22 @@ paths:
             schema:
               $ref: '#/components/schemas/PipelineExportBodyPost'
       responses:
-        200:
+        201:
```

Comment:
Yes, I think a `201` is correct here since the "resource created" is the export file.